src/apns_connection.erl

```diff
@@ -496,7 +504,8 @@ push(GunConn, DeviceId, HeadersMap, Notification, Timeout) ->
         {ok, Body} = gun:await_body(GunConn, StreamRef, Timeout),
         DecodedBody = jsx:decode(Body),
         {Status, ResponseHeaders, DecodedBody};
-      {error, timeout} -> timeout
+      {error, timeout} -> timeout;
+      {error, {closed, _}} -> closed
```

Comment:
What's this change about?

Reply:
This was intended to match specific error tuple when the process fails and not timed out. I will remove it